static/src/stylesheets/module/preferences/_notifications.scss

```diff
@@ -1,20 +1,239 @@
+    .notifications__button--#{$state} {
+        .live-notifications__label--#{$state} {
+            display: inline;
+        }
+    }
 }
 
-.notifications-link {
-    margin-top: $gs-baseline;
-    margin-bottom: $gs-baseline / 2;
+@include notifcations-button-style();
+@include notifications-button-toned(live, $live-default, $neutral-8);
+@include notifications-button-toned(dead, $live-default, $neutral-8);
+
+.live-notifications-denied {
+    position: fixed;
+    display: block;
+    top: .625rem;
+    left: 5.275rem;
+    width: 26.825rem;
+    height: 4.7625rem;
+    background: fade-out($live-default, 1 - .95);
+    transition: background-color .25s ease;
+    filter: alpha(opacify=50);
+    -moz-opacity: .5;
+    -khtml-opacity: .5;
+    z-index: 15000;
+    color: $neutral-8;
+
+    .button {
+        padding: 0;
+        width: $live-notifications-icon-width;
+        height: $live-notifications-icon-width;
+        border-color: #ffffff;
+
+        @include mq(tablet) {
+            border-color: colour(live-mute);
+        }
 
-    .button--follow {
-        margin-bottom: $gs-baseline;
+        &:hover {
+            border-color: #ffffff;
+        }
     }
 }
 
+.live-notifications-denied::after {
+    content: '';
+    display: inline-block;
+    width: 0;
+    height: 0;
+    border-top: 0;
+    position: absolute;
+    left: 3%;
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid $live-default;
+    top: -.6225rem;
+}
+
+.live-notifications-explainer {
+
+    margin-top: ($gs-baseline / 3) * 4;
+    margin-bottom: $gs-baseline;
+    background: $live-default;
+    color: $neutral-8;
+    display: block;
+
+    .button {
+        padding: 0;
+        width: $live-notifications-icon-width;
+        height: $live-notifications-icon-width;
+        border-color: #ffffff;
+
+        @include mq(tablet) {
+            border-color: colour(live-mute);
+        }
+
+        &:hover {
+            border-color: #ffffff;
+        }
+    }
+}
+
+.live-notifications-explainer::after {
+    content: '';
+    display: inline-block;
+    width: 0;
+    height: 0;
+    border-top: 0;
+    position: absolute;
+    left: 5%;
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid $live-default;
+    top: 2.7rem;
+}
+
+.live-notifications-explainer-svg {
+    svg {
+        @include fs-textSans(3);
+    }
+}
+
+
+
+.live-notifications-denied__content,
+.live-notifications-explainer__content {
+    position: relative;
+    box-sizing: border-box;
+    padding: $gs-baseline 0 ($gs-baseline/3)*2 $gs-gutter;
+}
+
+.live-notifications-denied__header,
+.live-notifications-explainer__header {
+    position: relative;
+    padding-right: gs-span(1) + $gs-gutter * .5;
+
+    .inline-marque-36 {
+        position: absolute;
+        left: 0 - $live-notifications-icon-width - $gs-gutter / 2;
```

Comment:
I agree that it's subjective but I think - $live-notifications-icon-width - $gs-gutter / 2; is less readable. So tough